README.md

```diff
@@ -31,11 +31,13 @@ See also: https://github.com/vbauer/java-blogs
 * Cult of Android http://www.cultofandroid.com
 * Droid Life http://www.droid-life.com
 * FXP http://fxpblog.co
+* Hootsuite http://code.hootsuite.com/category/code/android/
 * Novoda http://novoda.com/blog
 * Phandroid http://phandroid.com
 * Planet Android http://www.planetandroid.com
 * Talk Android http://www.talkandroid.com
 * Team Android http://www.teamandroid.com
+* WeddingParty http://nerds.weddingpartyapp.com/tech/categories/android/
```

Comment:
Wedding party is a company
On 11-Aug-2015 1:45 am, "Vladislav Bauer" <notifications@github.com> wrote:

> In README.md
> <https://github.com/vbauer/android-blogs/pull/9#discussion_r36678568>:
>
> >  * Novoda http://novoda.com/blog
> >  * Phandroid http://phandroid.com
> >  * Planet Android http://www.planetandroid.com
> >  * Talk Android http://www.talkandroid.com
> >  * Team Android http://www.teamandroid.com
> > +* WeddingParty http://nerds.weddingpartyapp.com/tech/categories/android/
>
> Maybe it will be better to move this blog in the "Individuals"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vbauer/android-blogs/pull/9/files#r36678568>.
>